lua/luasnip/util/parser/init.lua

```diff
@@ -117,6 +117,7 @@ local function backticks_to_variable(body)
 
 	-- append remaining characters.
 	var_string = var_string .. body:sub(processed_to, -1)
+	var_string = var_string:gsub("\\`", "`")
```

Comment:
I think we'd only want to `gsub` occurences in the remaining characters, the escaped ``` ` ``` inside unescaped backticks are already replaced once.
(``` `echo '\\\` lel'` ``` would become ``` echo '`lel' ```, instead of ``` echo '\`lel' ``` (I think so at least, please verify :sweat_smile:))

So basically, `body:sub(...):gsub("\\`", "`")` should work instead.

Reply:
Done